<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEGWnd</class>
 <widget class="QWidget" name="EEGWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>794</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="PlotWidget" name="plotwidget1" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget2" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget3" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget4" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget5" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget6" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget7" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget8" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget9" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget10" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget11" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget12" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget13" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget14" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget15" native="true"/>
     </item>
     <item>
      <widget class="PlotWidget" name="plotwidget16" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
